presto-main/src/main/java/com/facebook/presto/sql/planner/LogicalPlanner.java

```diff
@@ -275,26 +288,25 @@ private RelationPlan createInsertPlan(Analysis analysis, Insert insertStatement)
         plan = new RelationPlan(projectNode, scope, projectNode.getOutputSymbols());
 
         Optional<NewTableLayout> newTableLayout = metadata.getInsertLayout(session, insert.getTarget());
+        TableStatisticsMetadata statisticsMetadata = metadata.getInsertStatisticsMetadata(session, insert.getTarget());
```

Comment:
Apparently you want it to be consistent with the `newTableLayout`. But honestly i don't think that's a good name, since it is returned by the `getInsertLayout` method.